packages/panzer/adapters-stk/test/stk_connmngr/tEpetraLinearObjFactory.cpp

```diff
@@ -132,6 +132,8 @@ TEUCHOS_UNIT_TEST(tEpetraLinearObjFactory, buildTest_quad)
    Teuchos::RCP<Epetra_CrsGraph> graph = laFactory.getGraph(0,0);
    Teuchos::RCP<Epetra_CrsGraph> gGraph = laFactory.getGhostedGraph(0,0);
 
+   gGraph->Print(out);
+
```

Comment:
I take it this print statement was a leftover from debugging? Might want to remove.

Reply:
Unless you have strong opinions otherwise, I'm going to leave this in. It doesn't really cost anything extra in the tests.

Reply:
No strong opinion. The only thing it does is bloat the size of the saved output from unit tests. But its a pretty small sized test. Probably would not want to do this for something larger like the mini-em tests.

Reply:
I'll try and remember to take it out whenever we understand the bug in the filtering test.